api/src/main/java/io/strimzi/api/kafka/model/kafka/listener/KafkaListenerAuthenticationOAuth.java

```diff
@@ -217,6 +218,16 @@ public void setClientAudience(String audience) {
         this.clientAudience = audience;
     }
 
+    @Description("The grant type to use when making requests to the authorization server's token endpoint. Used for inter-broker authentication and for configuring OAuth 2.0 over PLAIN using the `clientId` and `secret` method.")
```

Comment:
This is incorrect isn't it, since the inter-broker authentication is always mTLS? Also does this default to `client_credentials` as well? If so we should state that.